src/charm.py

```diff
@@ -353,6 +418,44 @@ def _get_tls_config(self) -> dict:
             }
         }
 
+    def _get_tracing_config(self) -> dict:
+        """Return dictionary with opentelemetry configuration if available."""
+        # ref: https://doc.traefik.io/traefik/ma<woke>ster/observability/tracing/opentelemetry/
+        #  note: remove the <woke> tag
```

Comment:
@ru-fu pointed out that woke supports [inline excludes](https://docs.getwoke.tech/ignore/#in-line-and-next-line-ignoring)

```suggestion
        # wokeignore:rule=master
        # ref: https://doc.traefik.io/traefik/master/observability/tracing/opentelemetry/
```